encryption.js

```diff
@@ -116,13 +121,13 @@ function decrypt(passphrase, encryptedContent) {
     .then(decryptionKey => {
       return crypto.subtle.decrypt(
         {name: 'AES-GCM',
-         iv: parts.iv},
+          iv: parts.iv},
```

Comment:
Sure, I can manually change those. Not sure if ESLint has a rule for that formatting level, but another cool module we can use to format the codebase is [**prettier**](https://github.com/prettier/prettier), if we want everything converted to AST and reformatted. It's probably a fairly large diff, so I won't add that to this PR, but once this lands I can do a separate PR to see if it is worth the effort.